daffodil-cli/src/test/scala/org/apache/daffodil/cliTest/TestBlob.scala

```diff
@@ -43,15 +40,15 @@ class TestBlob {
    * will need to generate the test file(s) using the gen_blob.py script. It can
    * be found in and should be run from the directory:
    *
-   * daffodil-cli/src/it/resources/org/apache/daffodil/CLI/input/
+   * daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/
```

Comment:
Nothing under src/test/resources changed directories. Only scala code packages. 
The resources weren't all chopped up into little packages.

Reply:
Did you see the comment I wrote under my +1?  I suggested you rename CLI to cli or cliTest, although I wouldn't worry about it since the PR's merged already:

+1 with requested changes

I do not like cliTest as a package / folder name but Daffodil tests never get packaged and deployed so I won't object. However, please consider renaming CLI as a package / folder name since good Scala style uses all lowercase:

daffodil-cli/src/test/resources/org/apache/daffodil/CLI/* -> cli/* (or cliTest for consistency)
daffodil-cli/src/test/scala/org/apache/daffodil/CLI/Util.scala -> cliTest/Util.scala (like the rest)